package.json

```diff
@@ -6,13 +6,22 @@
   "license": "LGPL 2+",
   "main": "dist/image-widgets.js",
   "devDependencies": {
+    "angular": "^1.6.6",
```

Comment:
Do we have to pull these in as deps? Or can we load them out of bower like we do now.